src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Server-side Sampling
+sidebar_order: 1
+description: "Learn about how you can control which transactions Sentry retains."
+---
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in joining this limited release, please reach out at [abc@sentry.io](mailto:abc@sentry.io).
+
+</Note>
+
+Capturing a single trace involves minimal overhead, but capturing traces for every page load or every API request may add an undesirable load to your system. Sampling your events allows you to better manage the number of events sent to Sentry, so you can tailor your volume to your organization's needs. _Server-side sampling_ lets you control which transactions Sentry retains by setting sampling rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In contrast to [client-side sampling](/platform-redirect/?next=/configuration/sampling/), server-side sampling allows you to manage your sampling configuration centrally and lets you make changes without the need to reconfigure the SDK and redeploy your application.
```

Comment:
> This first paragraph still doesn't answer the "why" completely.
> As a user, I might end up here and may not know why I should care about this in the first place.
> Sampling mainly exists to balance data usage and data fidelity so that you get everything that matters without storing too many likely redundant events.
> 
> I think that what we have in the [JavaScript docs](https://docs.sentry.io/platforms/javascript/configuration/sampling/#sampling-transaction-events) captures this pretty well.

@danielkhan & @ale-cota - I've tried to update this per the JS docs. Feel free to tweak.